hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSInputStream.java

```diff
@@ -1003,7 +1003,7 @@ private LocatedBlock refetchLocations(LocatedBlock block,
       String description = "Could not obtain block: " + blockInfo;
       DFSClient.LOG.warn(description + errMsg
```

Comment:
Here we can see the Warning message on client side, is it?

Reply:
Thanks @ferhui for your comments. In some cases, such as Presto, only show exception msg on the web UI. If we want to know those dn info, we need to look for the log file, It's a bit of a hassle.

Reply:
Got it, it is reasonable.

Reply:
Thanks for your review.